lib/matplotlib/tests/test_collections.py

```diff
@@ -396,6 +396,7 @@ def test_EllipseCollection():
 @image_comparison(['polycollection_close.png'], remove_text=True, style='mpl20')
 def test_polycollection_close():
     from mpl_toolkits.mplot3d import Axes3D  # type: ignore
+    plt.rcParams['axes3d.automargin'] = True
```

Comment:
If these images are still regenerated, I would probably drop these lines? (Somehow, the only point in keeping them is to avoid having to regenerate the images.)